### DISCUSSION TITLE
Welcome to flutter_rich_text_editor Discussions!

### DISCUSSION THREAD

**Discussion by slightfoot** [Flutter-Bounty-Hunters/super_editor#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kherel** [comment]
Right now I'm using draft.js based editor on web. I'm trying to make something like Notion. The result is in json format, that is very easy to use in mobile flutter app. I like that everything is in blocks, so it's not hard to add customise block for media, or any other custom data. 

The problem with draft.js, is that it uses immutable.js + there are a lot of bugs and seems like the team are not goint do much to fix it.

There are anothe good project called react-contenteditable it's looks good, but it dosn't have good selectible api. So seems like it will take hunderds of hours to make my final goal come true.

If we could create something like notion editor in flutter, It would be great.

**Comment by klausries** [comment]
I am building an app to generate simple mobile first presences (flutter app based and HTML/AMP based) but I also need a half way real text editor. I am currently tinkering with:
* Zefyr -- seems a little hacked up but I kindof like the idea of quill
* https://github.com/sunderee/FlutterMCE -- very basic but maybe that's where I start
* reuse some JS component

Some aspects may also tie in well with superlist. So it depends on your product vision and timeline.
I might try to start with Zefyr and see if I can get it working, I tend to prefer co-investing.
